web-components/src/components/avatar/scss/avatar.scss

```diff
@@ -306,6 +312,21 @@
   display: inline-block;
   line-height: 0rem;
   vertical-align: middle;
+  --icon-color-channel-sms-inbound: var(--avatar-letter-bg-color);
+  --icon-color-channel-sms-outbound: var(--avatar-letter-bg-color);
+  --icon-color-channel-email-inbound: var(--avatar-violet-bg-color);
+  --icon-color-channel-email-outbound: var(--avatar-violet-bg-color);
+  --icon-color-channel-call: var(--avatar-green-bg-color);
+  --icon-color-channel-callback: var(--avatar-green-bg-color);
+  --icon-color-channel-headset: var(--avatar-pink-bg-color);
+  --icon-color-channel-campaign: var(--avatar-pink-bg-color);
+  --icon-color-channel-chat: #1170CF;
+  --icon-color-channel-emoji:#1170CF;
+  --icon-color-suspected-spam: #A65E00;
+  --icon-color-light-gray: #F7F7F7;
+  --icon-color-dark-gray: #E6E6E6;
+  --icon-color-white: #FDFDFD;
+  --icon-color-beige: #FFEBC2;
 }
```

Comment:
are these used? momentumBrandVisuals have the colour as part of the SVG don't think we need to do colouring unless its an icon

Did you test in dark mode?

We should never use hex colours every colour referenced should be a colour token i.e. icon-color-suspected-spam should use the momentum-design/tokens colours defined in the design